<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LibraryUi</class>
 <widget class="QMainWindow" name="LibraryUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>学生页面</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Mypicture.qrc">
    <normaloff>:/new/prefix3/SEU6.gif</normaloff>:/new/prefix3/SEU6.gif</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>1200</width>
     <height>600</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>200</width>
         <height>600</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>1</y>
          <width>200</width>
          <height>600</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="Mypicture.qrc">:/new/prefix3/SEUImage1.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QFrame" name="frame_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>200</width>
          <height>600</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>200</width>
           <height>100</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>100</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="Mypicture.qrc">:/new/prefix3/SEU6.gif</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QFrame" name="frame_4">
         <property name="geometry">
          <rect>
           <x>-1</x>
           <y>49</y>
           <width>201</width>
           <height>571</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QPushButton" name="pushButton_1">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>80</y>
            <width>200</width>
            <height>80</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;方正姚体&quot;;</string>
          </property>
          <property name="text">
           <string>个人信息及预约</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>200</y>
            <width>200</width>
            <height>80</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 20pt &quot;方正姚体&quot;;</string>
          </property>
          <property name="text">
           <string>当前状态</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>320</y>
            <width>200</width>
            <height>80</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 20pt &quot;方正姚体&quot;;</string>
          </property>
          <property name="text">
           <string>书籍推荐</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_4">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>440</y>
            <width>200</width>
            <height>80</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 20pt &quot;方正姚体&quot;;</string>
          </property>
          <property name="text">
           <string>丢失申报</string>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>0</y>
         <width>1001</width>
         <height>601</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #D8F4FF, stop: 1 #C0FFEC);</string>
       </property>
       <widget class="QWidget" name="page">
        <widget class="QFrame" name="frame_5">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1000</width>
           <height>600</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #D8F4FF, stop: 1 );</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QTableView" name="tableView">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>0</y>
            <width>801</width>
            <height>101</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">alternate-background-color: rgb(85, 170, 0);</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>140</y>
            <width>511</width>
            <height>51</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #D8F4FF, stop: 1 #C0FFEC);</string>
          </property>
         </widget>
         <widget class="QComboBox" name="comboBox">
          <property name="geometry">
           <rect>
            <x>640</x>
            <y>150</y>
            <width>151</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #D8F4FF, stop: 1 #C0FFEC);</string>
          </property>
         </widget>
         <widget class="QTableView" name="tableView_2">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>260</y>
            <width>931</width>
            <height>301</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_6">
          <property name="geometry">
           <rect>
            <x>660</x>
            <y>210</y>
            <width>111</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color:white;
border:none;
font-size:20px;
font-family:&quot;微软雅黑&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(51, 153, 255,1), stop:1 rgba(52, 246, 177,1));
padding-top:5;
padding-bottom:5;</string>
          </property>
          <property name="text">
           <string>查询</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_reserve">
          <property name="geometry">
           <rect>
            <x>840</x>
            <y>207</y>
            <width>93</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color:white;
border:none;
font-size:20px;
font-family:&quot;微软雅黑&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(51, 153, 255,1), stop:1 rgba(52, 246, 177,1));
padding-top:5;
padding-bottom:5;</string>
          </property>
          <property name="text">
           <string>预约</string>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="page_2">
        <widget class="QTableView" name="tableView_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>80</y>
           <width>941</width>
           <height>321</height>
          </rect>
         </property>
        </widget>
        <widget class="QFrame" name="frame_7">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>450</y>
           <width>541</width>
           <height>57</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="styleSheet">
             <string notr="true">color:white;
border:none;
font-size:20px;
font-family:&quot;微软雅黑&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(51, 153, 255,1), stop:1 rgba(52, 246, 177,1));
padding-top:5;
padding-bottom:5;</string>
            </property>
            <property name="text">
             <string>续借</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_7">
            <property name="styleSheet">
             <string notr="true">color:white;
border:none;
font-size:20px;
font-family:&quot;微软雅黑&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(51, 153, 255,1), stop:1 rgba(52, 246, 177,1));
padding-top:5;
padding-bottom:5;</string>
            </property>
            <property name="text">
             <string>查询</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="page_3">
        <widget class="QTableView" name="tableView_4">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>170</y>
           <width>751</width>
           <height>311</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>370</x>
           <y>80</y>
           <width>161</width>
           <height>71</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 24pt &quot;方正姚体&quot;;</string>
         </property>
         <property name="text">
          <string>书籍推荐</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_4">
        <widget class="QFrame" name="frame_6">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>180</y>
           <width>751</width>
           <height>121</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>请输入书籍条码号</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_2"/>
          </item>
         </layout>
        </widget>
        <widget class="QPushButton" name="pushButton_8">
         <property name="geometry">
          <rect>
           <x>350</x>
           <y>360</y>
           <width>211</width>
           <height>37</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color:white;
border:none;
font-size:20px;
font-family:&quot;微软雅黑&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(51, 153, 255,1), stop:1 rgba(52, 246, 177,1));
padding-top:5;
padding-bottom:5;</string>
         </property>
         <property name="text">
          <string>书籍报失</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_5"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="Mypicture.qrc"/>
 </resources>
 <connections/>
</ui>
